launch_scripts/eval_downstream.py

```diff
@@ -18,7 +18,11 @@
 
 
 def main():
-    prepare_torchrun_environment()
+    # enable run on a single GPU
+    try:
+        prepare_torchrun_environment()
+    except:
+        log.warning("Failed to prepare torchrun environment, continuing anyway")
```

Comment:
What is this case for?

Reply:
ahh, I think I added it to make it work with a single GPU,\